src/nevermined/Nfts.ts

```diff
@@ -512,4 +513,31 @@ export class Nfts extends Instantiable {
             throw Error(`Error listing ${ddo.shortId()} on secondary markets`)
         }
     }
+
+    public async lockBuyersPayments(
```

Comment:
```suggestion
    public async buySecondaryMarketNft(
```
this method will mask the whole `buy` flow, so name it accordingly :D